content/maxwell3_fdem/emi_tutorial/amplitudeandphase.rst

```diff
@@ -0,0 +1,129 @@
+
+.. math::
+	H^p_2(t) = K_{12} i^p(t),
+
+where :math:`i^p = I^p cos(\omega t)` and :math:`K_{12}` is coefficint depending on the geomtry of the system, the area, and number of turns of the primary coil, and attenuation of the wave between Loop1 and Loop2.
+
+Effectively, through Faraday's law, :math:`\mathcal{E} = -\imath \omega \Phi^p`, volatage will be generated, which generates induced currents (this has been done in :ref:`induced_currents_body`):
+
+.. math::
+    I(\omega) = \frac{-\imath \alpha}{1+\imath\alpha}\frac{\Phi^p}{L},
+
+where :math:`\alpha=\frac{\omega L}{R}`. EMF induced here can be written as
+
+.. math::
+	\mathcal{E} = -M\frac{d i^p}{dt} = -M \imath \omega I^p = -M \imath \omega \frac{H^p_2}{K_{12}},
+	:label: EMF
+
+where :math:`M` is mutual inductance, and this property illustrates how much voltage induced in Loop2 due to time varying current in Loop1. This yields :math:`\Phi^p = \frac{\mathcal{E}}{-\imath \omega} = M\frac{H^p_2}{K_{12}}`. The secondary magnetic field, :math:`H_s` at the Rx (Loop3) due to this current will be
+
+.. math::
+	H^s =K_{23} I(\omega)
+	= -K_{23}\frac{\imath \alpha}{1+\imath\alpha}\frac{\Phi^p}{L}
+	= -\frac{K_{23}M H^p_2}{K_{12}L}\frac{\imath \alpha}{1+\imath\alpha},
+	:label: secondaryHs
+
+where :math:`K_{23}` is similar coefficient with :math:`K`, but between body (Loop2) and Rx (Loop3).
+
+Eq. :eq:`secondaryHs` provides us an explicit form of secondary magnetic field measured at Loop3, but we are interested in the ration between :math:`H^s` and :math:`H^p`:
+
+.. math::
+	\frac{H^s}{H^{p}} = \frac{K_{23}}{K_{13}} \frac{I(\omega)}{I^p}
+	= -\frac{K_{23}M}{K_{13}L} \frac{\imath \alpha}{1+\imath\alpha}
+	= -\frac{K_{23}M}{K_{13}L} \frac{\alpha^2+\imath \alpha}{1+\alpha^2}
+	:label: HsHpratio
+
+Because the primary field at the Loop3 is
+
+.. math::
+	H^p = K_{13}I^p = K_{13}\frac{H^p_2}{K_{12}},
+
+where :math:`K_{13}` is similar coefficient with :math:`K`, but between body (Loop1) and Rx (Loop3). Eq. :eq:`HsHpratio` can be simplified as
+
+.. math::
+	\frac{H^s}{H^{p}} = -A Q(\alpha),
+
+where :math:`A = \frac{K_{23}M}{K_{13}L}` and response function, :math:`Q = \frac{\alpha^2+\imath \alpha}{1+\alpha^2}`. The amplitude can be written as
+
+.. math::
+	\Big| \frac{H^s}{H^p} \Big|
+	= \frac{K_{23}M}{K_{13}L} \frac{\alpha}{\sqrt{\alpha^2+1}} .
+
+.. note::
+
+	Because the ratio :math:`\frac{K_{23}M}{K_{13}L}` is generally very small, the ratio :math:`\frac{H^s}{H^{p}_3}` is small, regardless of the value of :math:`\alpha`. Often part per million (ppm) is used for the unit of this ratio.
+
+
+Phase of :math:`\frac{H^s}{H^p}`, :math:`\theta_s` will be same as that of :math:`I(\omega)`, hence
+
+.. math::
+	\theta_s = - \frac{\pi}{2} - \phi,
+	:label: phaseHs
+
+where
+
+.. math::
+	\phi=tan^{-1}(\frac{\omega L}{R})=tan^{-1}(\alpha).
+	:label: phaseHs_phi
+
+.. figure:: ./images/PhaseHs.png
```

Comment:
in this diagram, could you make the arrows showing the primary and secondary magnetic field blue and red (same as your diagrams showing field lines) - makes the connection immediate